neo4j/driver_with_context_examples_test.go

```diff
@@ -0,0 +1,82 @@
+package neo4j
+
+import (
+	"context"
+	"fmt"
+)
+
+var myDriver DriverWithContext
+var ctx context.Context
+
+func ExampleDriverWithContext_ExecuteQuery() {
+	results, err := myDriver.ExecuteQuery(ctx, "RETURN $value AS val", map[string]any{"value": 42})
```

Comment:
```suggestion
	result, err := myDriver.ExecuteQuery(ctx, "RETURN $value AS val", map[string]any{"value": 42})
```
maybe. As it's a single `EagerResult`.

Reply:
fixed in 5020b8e7b66285f8b3fb62f67c45b7b7c4043ad8